public/apps/configuration/utils/action-groups-utils.tsx

```diff
@@ -71,9 +71,16 @@ function getIndexSinglePermissions(): ActionGroupListingItem[] {
   }));
 }
 
-export async function getAllPermissionsListing(http: HttpStart): Promise<ActionGroupListingItem[]> {
-  const actionGroups = await fetchActionGroupListing(http);
-  return actionGroups.concat(getClusterSinglePermissions()).concat(getIndexSinglePermissions());
+export async function getAllPermissionsListing(http: HttpStart): Promise<PermissionListingItem[]> {
+  return getAllPermissionsListingLocal(await fetchActionGroups(http));
+}
+
+export async function getAllPermissionsListingLocal(
```

Comment:
Minor: since there is no network call and it just merges the data, probably call it mergeAllPermissions

Reply:
Agreed.